src/wp-admin/includes/class-wp-upgrader.php

```diff
@@ -248,21 +248,34 @@ public function fs_connect( $directories = array(), $allow_relaxed_file_ownershi
 	 * @param string $package          The URI of the package. If this is the full path to an
 	 *                                 existing local file, it will be returned untouched.
 	 * @param bool   $check_signatures Whether to validate file signatures. Default false.
+	 * * @param array $args {
+	 *     Optional. Array of arguments for downloading a package. Default empty array.
+	 *
+	 *     @type array  $hook_extra                  Extra arguments to pass to the filter hooks. Default empty array.
```

Comment:
The variable name is mismatched with the one defined below. How about we name this `$extra_args`?